azkaban-web-server/src/main/java/azkaban/imagemgmt/servlets/ImageTypeServlet.java

```diff
@@ -116,7 +131,7 @@ protected void handlePost(
       // prepare to send response
       resp.setStatus(HttpStatus.SC_CREATED);
       resp.setHeader("Location",
-          IMAGE_TYPE_WITH_ID_URI_TEMPLATE.createURI(imageTypeId.toString()));
+          IMAGE_TYPE_WITH_NAME_URI_TEMPLATE.createURI(imageType));
```

Comment:
Same comment as above, using the name of an image type instead of its ID in the URI to obtain resource details forces the name to be unique. How are we planning to support renames if a user made a typo while creating the image type or the name of the jobtype changed.

Reply:
The name needs to be unique. Renames should be possible with a post/put request.